ONSdigital/ssdc-rm-support-tool
# Updated case details page

# Motivation and Context
Add some names to case details page

# What has changed
Added the names

# How to test?
Run it locally can you see the collection Exercise name and the Survey Name.  great
Note: This area could do with refactoring to a nicer pattern where just the CaseId is passed and the routing is nicer

# Links
https://trello.com/c/ML66qJfo/2702-spike-investigate-if-we-can-ingest-files-which-include-boms-1-day

# Screenshots (if appropriate):
